uniswap_swap_detection/src/agent.spec.ts

```diff
@@ -0,0 +1,244 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  ethers,
+} from "forta-agent";
+import { utils, Contract } from "ethers";
+
+import agent, { provideHandleTransaction } from "./agent";
+import { SWAP_EVENT } from "./constant";
+import { createAddress } from "forta-agent-tools";
+import {
+  MockEthersProvider,
+  TestTransactionEvent,
+} from "forta-agent-tools/lib/test";
+
+const iface: utils.Interface = new utils.Interface([
+  "function token1() external view returns (address)",
+  "function token0() external view returns (address)",
+  "function fee() external view returns (uint24)",
+]);
+
+export const CREATED_POOL =
+  "event PoolCreated(address indexed token0,address indexed token1,uint24 indexed fee,int24 tickSpacing,address pool)";
+describe("swap occur", () => {
```

Comment:
```suggestion
  "event PoolCreated(address indexed token0,address indexed token1,uint24 indexed fee,int24 tickSpacing,address pool)";
  
describe("swap occur", () => {
```
Small detail, but spacing between declarations are important to make it easier to read.

Also, suggestion to rename test suite to something like
`describe("UniswapV3 Swap Detection Test Suite", ... );`